python/lsst/ctrl/oods/msgQueue.py

```diff
@@ -38,21 +38,28 @@
 
 
 class MsgQueue(object):
-    """Report on new messages
+    """Read messages from Kafka
 
     Parameters
     ----------
-    config: `dict`
+    brokers : `dict`
```

Comment:
If possible, please specify the key and value types here and in other container types.

Reply:
Fixed